tutorials/14-food-review-tone-analysis-and-response.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip uninstall -y evadb
</pre><p>Can we clean this up? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
You meant just remove this?

Reply:
Yeah. Why we need to uninstall EvaDB first?